modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,483 @@
+
+gcloud compute addresses create <address-name>
+
+gcloud compute routers nats create <nat-config-name> \
+   --project <host-project-id> \
+   --router <router-name> \
+   --region <region> \
+   --nat-all-subnet-ip-ranges \
+   --nat-external-ip-pool <address-name> \
+   --enable-endpoint-independent-mapping
+```
+
+. Create VPC firewall rules.
++
+** Redpanda ingress:
++
+```bash
+gcloud compute firewall-rules create redpanda-ingress \
+  --description="Allow access to Redpanda cluster" \
+  --network="<vpc-name>" \
+  --project="<host-project-id>" \
+  --direction="INGRESS" \
+  --target-tags="redpanda-node" \
+  --source-ranges="10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,100.64.0.0/10" \
+  --allow="tcp:9092-9094,tcp:30081,tcp:30082,tcp:30092"
+```
++
+** Master webhooks:
++
+```bash
+gcloud compute firewall-rules create gke-redpanda-cluster-webhooks \
+  --description="Allow master to hit pods for admission controllers/webhooks" \
+  --network="<vpc-name>" \
+  --project="<host-project-id>" \
+  --direction="INGRESS" \
+  --source-ranges="<gke-master-cidr-range>" \
+  --allow="tcp:9443,tcp:8443,tcp:6443"
+```
+
+== Configure the service project
+
+. Enable Google APIs in the service project:
++
+```bash
+gcloud services enable cloudresourcemanager.googleapis.com --project <service-project-id>
+gcloud services enable dns.googleapis.com --project <service-project-id>
+gcloud services enable secretmanager.googleapis.com --project <service-project-id>
+gcloud services enable compute.googleapis.com --project <service-project-id>
+gcloud services enable iam.googleapis.com --project <service-project-id>
+gcloud services enable storage-api.googleapis.com --project <service-project-id>
+gcloud services enable container.googleapis.com --project <service-project-id>
+gcloud services enable serviceusage.googleapis.com --project <service-project-id>
+```
+
+. Create storage buckets at the service project in the same region as the cluster:
++
+```bash
+gcloud storage buckets create gs://<tiered-storage-bucket-name> \
+  --location="<region>" \
+  --uniform-bucket-level-access
+
+gcloud storage buckets create gs://<management-storage-bucket-name> \
+  --location="<region>" \
+  --uniform-bucket-level-access
+
+gcloud storage buckets update gs://<management-storage-bucket-name> --versioning
+``` 
++
+* Redpanda uses the Tiered Storage bucket for writing log segments. This should not be versioned.
+* Redpanda uses the Management Storage bucket to store cluster metadata. This can have versioning enabled.
+
+. Create service accounts at the service project for the Redpanda agent, the Redpanda cluster, the Redpanda GKE cluster, Redpanda Console, and Redpanda connectors. 
+
+.. Redpanda agent SA:
++
+```bash
+gcloud iam service-accounts create redpanda-agent \
+  --display-name="Redpanda Agent Service Account"
+```
+
+* Redpanda agent custom role permissions:
++
+```bash
+gcloud iam roles create redpanda_agent_role \
+  --project="<project-id>" \
```

Comment:
Change `<project-id>` to `<service-project-id>` so it will fill in the variable used above

Reply:
Actually, the same goes for all project IDs from here on down